pages/to_do_list_page.ts

```diff
@@ -0,0 +1,43 @@
+import { test, Page, expect } from '@playwright/test';
+
+export class ToDoListPage {
+    readonly page: Page;
+
+    constructor(page: Page) {
+        this.page = page;
+    }
+    
+
+    async open() {
+        await this.page.goto('https://webdriveruniversity.com/To-Do-List/index.html');
+    }
+
+    async clickPlusButton(){
+        await this.page.click("#plus-icon");
+    }
+
+    async addNewToDo(newToDo: string) {
+        await test.step ('Add new TODO: \"' + newToDo + '\" to the TODO list.', async () => {
+            if (await this.page.getByPlaceholder('Add new todo').isHidden()){
+                await this.clickPlusButton();
+            }
+            await this.page.getByPlaceholder('Add new todo').focus();
+            await this.page.keyboard.type(newToDo);
+            await this.page.keyboard.press("Enter");
+        })
+    }
+
+    async writeAndReturnHowManyItemsInToDo() {
+        let numberOfItems = await this.page.locator('li').count();
+        console.log("On the list are " + numberOfItems.toString() + " elements.");
```

Comment:
1. don't use console.log. You should use test.step if you want add string to report. 
2. don't use + -> check how works `${}` in typescript

Reply:
resolved